pulp_rpm/app/models.py

```diff
@@ -269,6 +270,7 @@ def to_createrepo_c(self):
             createrepo_c.Package: package itself in a format of a createrepo_c package object
 
         """
+
```

Comment:
There's a conflict here between black and flake8-docstrings. Black adds this line while flake8-docstrings enforces a rule that there should be no blank line after docstrings (D202).

Reply:
I am ignoring D202 for now until https://github.com/PyCQA/pydocstyle/issues/361#issuecomment-498625056 is addressed.